### PR TITLE
[FIX] export

### PR DESCRIPTION
**Pull request by nim-odoo**

### CODE REVIEW

**Comment by nim-odoo** [comment]
robodoo rebase-ff

File: odoo/addons/test_impex/tests/test_export.py [hunk 1]
```diff
@@ -243,19 +243,22 @@ def test_empty(self):
...
             self.export('2011-11-07 21:05:48'),
             [[u'2011-11-07 21:05:48']])
 
     def test_tz(self):
-        """ Export ignores the timezone and always exports to UTC
+        """ Export converts the value in the user's TZ
 
         .. note:: on the other hand, export uses user lang for name_get
         """
         # NOTE: ignores user timezone, always exports to UTC
```
Comment (odony) [inline odoo/addons/test_impex/tests/test_export.py]: this comment should probably be updated/deleted too

File: odoo/addons/test_impex/tests/test_export.py [hunk 2]
```diff
@@ -291,7 +294,7 @@ def test_localized_export(self):
             })
         self.assertEqual(
             self.export(2, context={'lang': 'fr_FR'}),
-            [[u'Bar']])
+            [[u'titi']])
```
Comment (odony) [inline odoo/addons/test_impex/tests/test_export.py]: ooh, nice catch

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**